src/analytics/config.py

```diff
@@ -1,25 +1,22 @@
 import os
-
 from datetime import datetime
 from pathlib import Path
-from dotenv import load_dotenv
+
 from decouple import config as env_var
+from dotenv import load_dotenv
 from flasgger import LazyString
 
 env_path = Path(".") / ".env"
 load_dotenv(dotenv_path=env_path, verbose=True)
 
 TWO_HOURS = 7200  # seconds
 
-API_V2_BASE_URL = "/api/v2/analytics"
+API_BASE_URL = "/api/v2/analytics"
```

Comment:
Thanks @Mnoble-19 for this. Please consider the possibility of having more than one API version running in this microservice. Consider a scenario where we have V1, V2, V3, V4 , etc....all running and available for this micro, how would it be possibly organised in our codebase?

Reply:
@Mnoble-19 checkout blueprints. They are quite easy to use. I can support.

Reply:
Alright @NicholasTurner23 please remember to coordinate this in another PR. Thanks for volunteering.